mirrord/layer/src/tcp.rs

```diff
@@ -69,6 +71,7 @@ impl From<&Listen> for SocketAddr {
 }
 
 pub(crate) trait TcpHandler {
+    const IS_STEAL: bool;
```

Comment:
We don't hold `TcpHandler` as `Box<dyn TcpHandler>` anywhere right?

This could be turned in a const generic parameter on `TcpHandler` (I think).

Not worth going into this rabbit hole though, just something to dream about.

Reply:
Like this? https://github.com/metalbear-co/mirrord/pull/1557/commits/7983ee5a17ba7b4911b246c619541e62ed4543b4

Reply:
Yup. Thought this would require more changes, but this trait is not very "shared" I guess.

`const`fy-all the things!